package.json

```diff
@@ -41,6 +41,7 @@
         "@web3-react/injected-connector": "^6.0.7",
         "@web3-react/walletconnect-connector": "6.2.4",
         "@web3-react/walletlink-connector": "6.2.3",
+        "@webflow/webflow-cli": "^1.5.1",
```

Comment:
Should this be a dev dependency?

Reply:
Couldn't find a reference on this in documentation, but the command I used to install it, installed it in a dependencies. But I can change it manually I guess.